### PR TITLE
Update protocol version, shift to julia v0.6

### PR DESCRIPTION
**Pull request by ZacLN**

### CODE REVIEW

File: scripts/languageserver/packages/AbstractTrees [hunk 1]
```diff
@@ -1 +0,0 @@
-Subproject commit 74695d4634c2286e20c75accb2739d07344fe56a
```
Comment (davidanthoff) [inline scripts/languageserver/packages/AbstractTrees]: Something is wrong here with the submodules, this shouldn't reference local paths from your machine... Also, when I checkout this branch all the submodules don't update correctly, which I assume is due to whatever is going on here...

Comment (ZacLN) [inline scripts/languageserver/packages/AbstractTrees]: Hmm, yep that's all wrong. I'll have a fiddle

Comment (ZacLN) [inline scripts/languageserver/packages/AbstractTrees]: OK I've fixed it back to proper-submodules. I'll set the specific commits after the LS PR is merged/ the Julia release

**Comment by ZacLN** [comment]
This is a mess, I can't get the submodules sorted so I'm going to close this open a new one.

**Pull request closed without merging**